src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -51,7 +52,9 @@ public sealed class CompletionItem : IComparable<CompletionItem>
         /// The text used to determine the order that the item appears in the list.
         /// This is often the same as the <see cref="DisplayText"/> but may be different in certain circumstances.
         /// </summary>
-        public string SortText { get; }
+        public string SortText => _sortText ?? DisplayText;
```

Comment:
CompletionItems are supposed to be 'pure data', without logic.  I would prefer we not have this.